code/controllers/configuration/entries/game_options.dm

```diff
@@ -364,3 +364,5 @@
 /datum/config_entry/number/auto_transfer_delay
 	config_entry_value = 72000
 	min_val = 0
+
+/datum/config_entry/flag/dynamic_voting

```

Comment:
An admin "verb" should be added to switch on or off dynamic voting for a given round through CONFIG_SET (modified configs do not carry over successive rounds iirc). It'd be sad if this ended up backend only.

Reply:
I've been an admin but I'm not sure where that would go. Game panel? Secrets?

Reply:
I think somewhere easily accessible like "server" or "fun" tabs.
Since it currently would have to be hastily toggled before voting happens to work, it shouldn't be added to the secrets or game panels.
Though I suggest you ask @kevinz000, who is an administator, unlike me.

Reply:
Right, "server" tab makes most sense, I'll do it

Reply:
This sufficient? https://github.com/Citadel-Station-13/Citadel-Station-13/pull/9557/commits/da7d4345b5bdcaa07662de1ef5d9abe15329cb23

Reply:
Upon testing: it's sufficient.